app/cafe.py

```diff
@@ -0,0 +1,22 @@
+import datetime
+
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError
+        vaccine_info = visitor["vaccine"]
+        if vaccine_info["expiration_date"] < datetime.date.today():
+            raise OutdatedVaccineError
+        if not visitor.get("wearing_a_mask", True):
```

Comment:
Remove `True `as default value because If 'wearing mask' key is not specified in a visitor dictionaty, you will not receive a NotWearingMaskError.